src/components/atoms/SvgIcon/SvgIcon.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styles from './SvgIcon.scss';
+
+import icons from './icons';
+
+const SvgIcon = ({ iconType, tag }) => {
+    return React.createElement(`${tag}`, {
+        className: styles.root,
+        dangerouslySetInnerHTML: { __html: icons[iconType] },
+    });
+};
+
+SvgIcon.defaultProps = {
+    tag: 'span'
+}
+
+SvgIcon.propTypes = {
+    iconType: string,
```

Comment:
mark as `isRequired`. Unless a prop is truly an option it should always have a default or be required